composer.json

```diff
@@ -28,7 +28,7 @@
 		"phpseclib/phpseclib": "*",
 		"rackspace/php-opencloud": "v1.9.2",
 		"james-heinrich/getid3": "dev-master",
-		"ircmaxell/random-lib": "v1.0.0",
+		"ircmaxell/random-lib": "~1.1",
```

Comment:
This is the proper way.
The exact version is being locked in the composer.lock file.

We just have to take care, that on update people run `composer update vendor/package`, so they only update what they actually want to update.